src/encoded/static/components/experiment.js

```diff
@@ -19,12 +20,33 @@ import { BiosampleSummaryString, BiosampleOrganismNames, CollectBiosampleDocs, A
 import ViewControlRegistry, { ViewControlTypes } from './view_controls';
 
 
+/**
+ * 'Experiment' search results view-control filter. For most experiment searches, this just returns
+ * the default views for experiments. But if this displays the reference-epigenome matrix, it
+ * returns the subset of views relevant to those.
+ * @param {array} types Views defined by default for the Experiment @type.
+ * @param {object} results Current page's search-results object
+ *
+ * @return {array} Views that apply to the current search results.
+ */
+const viewControlFilter = (types, results) => {
+    let views;
+    const parsedUrl = url.parse(results['@id']);
+    if (parsedUrl.pathname === '/reference-epigenome-matrix/') {
+        views = ['Search', 'Report'];
+    } else {
+        views = types.filter(type => type !== results['@type'][0]);
+    }
+    return views;
+};
```

Comment:
I mentioned this problem in the QA notes in the ticket. My view-controls module has no consistent way to deal with this since I never expected the "type" to be different from what’s actually desired. This needs to be addressed, but I think in another ticket to make the view controls more flexible.